python/pecos/reps/pypmir/op_types.py

```diff
@@ -19,11 +19,13 @@ def __init__(
         args: list | None = None,
         returns: list | None = None,
         metadata: dict | None = None,
+        angles: tuple[list[float], str] | None = None,
     ) -> None:
         self.name = name
         self.args = args
         self.returns = returns
         self.metadata = metadata
+        self.angles = angles
```

Comment:
I don't think a general operation needs angles attached to them as that is something special to QOps. Is this sort of thing needed for the pydantic translation, or you would rather have the parent class have all possible fields?